classes/webservice/WebserviceOutputBuilder.php

```diff
@@ -35,7 +35,10 @@ class WebserviceOutputBuilderCore
     const VIEW_LIST = 1;
     const VIEW_DETAILS = 2;
 
-    protected $wsUrl;
+    public static $wsUrl;
+    public static $languages;
```

Comment:
```suggestion
    public static $wsUrl;
    public static $languages;
```

Could you add PHPDoc for these attributes ?

Reply:
Added in d3f57f3 - `$languages` has been removed since its unused here